crates/forge/tests/cli/build.rs

```diff
@@ -55,10 +55,14 @@ Compiler run successful!
 // tests build output is as expected
 forgetest_init!(build_sizes_no_forge_std, |prj, cmd| {
     cmd.args(["build", "--sizes"]);
-    let stdout = cmd.stdout_lossy();
-    assert!(!stdout.contains("console"), "\n{stdout}");
-    assert!(!stdout.contains("std"), "\n{stdout}");
-    assert!(stdout.contains("Counter"), "\n{stdout}");
+    cmd.assert_success().stdout_eq(str![
+        r#"
+...
+| Contract | Size (B) | Margin (B) |
+|----------|----------|------------|
+...
```

Comment:
Should include Counter

Reply:
Done, due to formatting can't easily add the `[..]` into the table due to whitespace formatting but I can add the `Counter` title.